test/test_utils.py

```diff
@@ -12,6 +14,8 @@
 )
 from tpm2_pytss.utils import *
 from tpm2_pytss.internal.templates import _ek
+
+from src.tpm2_pytss.utils import credential_to_tools, tools_to_credential
```

Comment:
This shouldn't be needed, it should already be imported at line 15

Reply:
So this breaks the tests against policy for unknown reasons, if I remove this line it works

Reply:
> So this breaks the tests against policy for unknown reasons, if I remove this line it works

Iv'e seen weird issues when you import from the `src.tpm2_pytss` package over the `tpm2_pytss` checks on types fail and probable other weird things ensue as the types get bound different packages:
```python3
>>> from tpm2_pytss.constants import ESYS_TR
>>> from src.tpm2_pytss.constants import ESYS_TR2
>>> ESYS_TR.NONE == ESYS_TR2.NONE
True
>>> ESYS_TR.NONE is ESYS_TR2.NONE
False
```

Reply:
Removed the imports :)